client/src/components/Header.js

```diff
@@ -26,8 +28,8 @@ const Header = ({ user }) => {
         </div>
       </div>
       <div className="navbar">
-        <div>
-          <i>Logo</i>
+        <div className="logo" onClick={() => history.push("/home")}>
+          <img src={Logo} alt="site logo" />
         </div>
```

Comment:
```suggestion
        <Link to="/home" className="logo">
          <img src={Logo} alt="site logo" />
        </Link>
```

react-router has the component `Link` that manages the routing for you, so you don't have to do `onClick`s etc. (https://reacttraining.com/react-router/web/api/Link)